patterns/1-initial/junkyard-styled-innersourcing.md

```diff
@@ -25,7 +28,9 @@ Two situations:
 * Provide some sort of grading (Alexander marked his patterns with 0, 1 or 2 stars depending upon how convinced he was that it was a good pattern), so that the reader can be aware of what to expect. But how to do that without insulting the contributor? (solution: engage them in the process -- have 2 scores, what they think, and what users think or how many times its been used)
 
 ## Status
-Brainstormed pattern idea in progress
+
+* Initial
+* Brainstormed pattern idea in progress
 
 ## Authors
 * Georg Grutter
```

Comment:
Since we're cleaning up: do we want to continue to mention affiliations? I can't think of a good reason, TBH.

Reply:
@gruetter what do you mean with "affiliations"? The company that the author of pattern is working for?

Reply:
I went through all other feedback and worked it in as best as I could.
Will merge the PR now but feel free to add a new issue for this related to your idea about the _affiliations_.